api/blockAPI_test.go

```diff
@@ -218,7 +288,45 @@ func TestBlockAPI(t *testing.T) {
 	b := NewBlockAPI(db, 1, pl, cfg)
 	blockObject, _ := blocksDecompress()
 	blockNumbers := getBlockNumbers(blockObject)
+	receiptDataNumber, receiptDataHash, err := blockReceiptsTransform()
+	if err != nil {
+		log.Error("Error returned from blockReceiptsTransform", "err", err)
+	}
 	for i, block := range blockNumbers {
+		blockNo := BlockNumberOrHashWithNumber(block)
```

Comment:
So there is nothing wrong technically here, in terms of organization, I would prefer if the tests appear roughly in the same order that they appear in the API files. And so this needs to be moved to the bottom of the blockNumbers loop.